modules/apps/data-engine/data-engine-taglib/src/main/resources/META-INF/resources/data_layout_builder/js/actions.es.js

```diff
@@ -102,19 +103,43 @@ export const dropLayoutBuilderField = ({
 };
 
 export const dropFieldSet = ({
+	availableLanguageIds = [],
 	dataLayoutBuilder,
 	fieldName,
-	fieldSet,
+	fieldSet: _fieldSet,
```

Comment:
1. Keven, this sounds strange when props are renamed only to be assigned to a variable that will have the same name as the props.

Reply:
This all commit changes belongs to use a fieldSet story, and for this reason will no longer be used by this PR!